index.md

```diff
@@ -0,0 +1,39 @@
+This site under construction. See <https://github.com/bazel-contrib/SIG-rules-authors/issues/2>
+
+We attempt to list many Bazel rulesets, and provide guidance about which ones are best to depend on.
+Modules listed on <https://registry.bazel.build> are periodically mirrored here.
+
+Other future datasources may include:
+
+- oss-compass, e.g. [report for rules_docker](https://oss-compass.org/analyze/snbnfub0)
+
+Definitions:
```

Comment:
Definitions here don't currently line up with what the table get populated with. Re-purpose to just define what the terms mean for OSS projects and create another section for stats that are not yet gather but could be in the future?